### DISCUSSION TITLE
How can I send an email OTP to enter an OTP on the Ionic HTML page using Ionic, Angular, and Firebase?

### DISCUSSION THREAD

**Discussion by ionicsiva** [ionic-team/capacitor#6935] [Ideas]
"I have the  question: How can I send an email OTP using Ionic, Angular, and Firebase? Like many others here, I want to create a system where an OTP is sent when an event occurs on the frontend. I've searched everywhere for a solution."

I'm working with Ionic, Angular, and the Firebase plugin for this.

#ionic7 #firebase #email #otp

**Comment by jcesarmobile** [comment]
closing as duplicate of https://github.com/ionic-team/capacitor/discussions/6934
